resources/ru/cemu.po

```diff
@@ -152,921 +1146,1090 @@ msgstr ""
+msgstr "&Активный аккаунт"
 
-#: MainWindow.cpp:2183
-msgid "&Experimental"
-msgstr "&Экспериментальный"
-
-#: MainWindow.cpp:2186
+#: MainWindow.cpp
 msgid "&Console region"
 msgstr "&Регион консоли"
 
-#: MainWindow.cpp:2187
+#: MainWindow.cpp
 msgid "&Console language"
 msgstr "&Язык консоли"
 
-#: MainWindow.cpp:2188
-msgid "&Options"
-msgstr "&Опции"
-
-#: MainWindow.cpp:2192
+#: MainWindow.cpp
 msgid "&Memory searcher"
 msgstr "&Поиск памяти"
 
-#: MainWindow.cpp:2194
+#: MainWindow.cpp
 msgid "&Tools"
 msgstr "&Инструменты"
 
-#: MainWindow.cpp:2199
+#: MainWindow.cpp
 msgid "&Single-core interpreter"
 msgstr "&Одноядерный интепретатор"
 
-#: MainWindow.cpp:2200
+#: MainWindow.cpp
 msgid "&Single-core recompiler (fast)"
 msgstr "&Одноядерный рекомпилятор (быстрый)"
 
-#: MainWindow.cpp:2201
+#: MainWindow.cpp
 msgid "&Dual-core recompiler (fast, unstable!)"
 msgstr "&Двухядерный рекомпилятор (быстрый, нестабильный)"
 
-#: MainWindow.cpp:2202
+#: MainWindow.cpp
 msgid "&Triple-core recompiler (fast, unstable!)"
 msgstr "&Трехядерный рекомпилятор (быстрый, нестабильный)"
 
-#: MainWindow.cpp:2206
-msgid "&Cycle based timer"
-msgstr "&Таймер основанный на цикле"
-
-#: MainWindow.cpp:2207
-msgid "&Host based timer (recommended)"
-msgstr "&Таймер основанный на хосте (рекомендуется)"
-
-#: MainWindow.cpp:2210
+#: MainWindow.cpp
 msgid "&Mode"
 msgstr "&Режим"
 
-#: MainWindow.cpp:2211
-msgid "&Timer"
-msgstr "&Таймер"
-
-#: MainWindow.cpp:2212
+#: MainWindow.cpp
 msgid "&CPU"
-msgstr "&ЦПУ"
+msgstr "&CPU"
 
-#: MainWindow.cpp:2216
+#: MainWindow.cpp
 msgid "&Scan NFC tag from file"
-msgstr "&Сканировать тег NFC из файла"
+msgstr "&Сканировать NFC-тег из файла"
 
-#: MainWindow.cpp:2217
+#: MainWindow.cpp
 msgid "&NFC"
 msgstr "&NFC"
 
-#: MainWindow.cpp:2223
+#: MainWindow.cpp
 msgid "&Unsupported API calls"
 msgstr "&Неподдерживаемые вызовы API"
 
-#: MainWindow.cpp:2224
-#, fuzzy
+#: MainWindow.cpp
 msgid "&Coreinit File-Access"
-msgstr "&Доступ к файлам ядра"
+msgstr ""
 
-#: MainWindow.cpp:2225
+#: MainWindow.cpp
 msgid "&Coreinit Thread-Synchronization API"
-msgstr "&API синхронизации потоков ядра"
+msgstr "&API синхронизации потоков Coreinit"
 
-#: MainWindow.cpp:2226
+#: MainWindow.cpp
 msgid "&Coreinit Memory API"
-msgstr "&API памяти ядра"
+msgstr "&API памяти Coreinit"
 
-#: MainWindow.cpp:2227
+#: MainWindow.cpp
 msgid "&GX2 API"
-msgstr "&GX2 API"
+msgstr ""
 
-#: MainWindow.cpp:2228
+#: MainWindow.cpp
 msgid "&Audio API"
-msgstr "&Аудио API"
+msgstr "&API Звука"
 
-#: MainWindow.cpp:2229
+#: MainWindow.cpp
 msgid "&Input API"
-msgstr "&Управление API"
+msgstr "&API Ввода"
 
-#: MainWindow.cpp:2230
+#: MainWindow.cpp
 msgid "&Socket API"
-msgstr "&Socket API"
+msgstr ""
 
-#: MainWindow.cpp:2231
+#: MainWindow.cpp
 msgid "&Save API"
-msgstr "&API сохранения"
+msgstr "&API Сохранений"
 
-#: MainWindow.cpp:2232
+#: MainWindow.cpp
 msgid "&H264 API"
-msgstr "&H264 API"
+msgstr ""
+
+#: MainWindow.cpp
+msgid "&NFP API"
+msgstr ""
```

Comment:
It's perfectly fine to translate these strings, just translate them all in a consistent way (some of them are missing)

Reply:
Should I change the sequence of words in untranslated items such as "H264 API" to match the translated ones?

Reply:
I'm also not sure about "Coreinit File-Access", what's "Coreinit" here?

Reply:
> Should I change the sequence of words in untranslated items such as "H264 API" to match the translated ones?

Yes, if it's not a grammatical error in your language.

> I'm also not sure about "Coreinit File-Access", what's "Coreinit" here?

Dunno tbh, leave "Coreinit" as it is. :D